frontend/src/pages/timepapercreate/CreateButton.jsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import style from './timePaperCreate.module.css';
+
+export default function Button({ children }) {
+  return <button className={`${style.buttonBasic}`}>{children}</button>;
```

Comment:
```suggestion
  return <button className=${style.buttonBasic}>{children}</button>;
```

이 구조로 작성하면 가독성이 좀 더 좋을 것 같습니다!